opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/wiremock/mocks/WiremockAccountInformationRequest.java

```diff
@@ -95,14 +95,13 @@ public SELF fintech_calls_list_accounts_for_anton_brueckner_no_ip_address() {
     public SELF user_anton_brueckner_provided_initial_parameters_but_without_psu_id_to_list_accounts_with_all_accounts_consent() {
         String body = readResource("restrecord/tpp-ui-input/params/unknown-user-all-accounts-consent.json");
 
-        ExtractableResponse<Response> response = RestAssured
+        ExtractableResponse<Response> response = withSignatureHeaders(RestAssured
```

Comment:
This function is called within the user session (from UI), not Fintech - it is protected only by users' cookie and not by request signing - no signature headers here

Reply:
Removed signature header adding for non-fintech requests